sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -1654,13 +1654,13 @@ public async Task UpdateMySqlDataFeedWithMinimumSetupAndGetInstance()
 
             await using var disposableDataFeed = await DisposableDataFeed.CreateDataFeedAsync(adminClient, dataFeedToCreate);
 
-            DataFeed dataFeedToUpdate = await adminClient.GetDataFeedAsync(disposableDataFeed.Id);
+            DataFeed dataFeedToUpdate = disposableDataFeed.DataFeed;
 
             dataFeedToUpdate.Description = description;
 
             DataFeed updatedDataFeed = await adminClient.UpdateDataFeedAsync(dataFeedToUpdate);
 
-            ValidateDataFeedWithMinimumSetup(updatedDataFeed, disposableDataFeed.Id, dataFeedName, description);
+            ValidateDataFeedWithMinimumSetup(updatedDataFeed, dataFeedName, description, dataFeedToUpdate.Id);
```

Comment:
The `expectedId` is an optional parameter now (used in Update tests, but not in Create tests), so I needed to move it.